CHANGELOG.md

```diff
@@ -14,6 +15,15 @@
 
 - Initial beta release.
 - Added support for `5.0.0-beta.*` versions of SKY UX component libraries. [#226](https://github.com/blackbaud/skyux-layout/pull/226)
+=======
```

Comment:
👇 Looks like you may have committed some merge conflicts.

Reply:
Yikes. Good catch. This has been fixed.